example/etcd-backup-operator/backup_cr.yaml

```diff
@@ -7,5 +7,4 @@ spec:
   storageType: S3
   s3:
     s3Bucket: jenkins-etcd-operator
```

Comment:
Make this a parameter `<s3-bucket-name>` since this is now an example and not just for internal use.